lib/image_optim/cmd.rb

```diff
@@ -44,6 +86,38 @@ def check_status!
 
         fail SignalException, status.termsig
       end
+
+      def init_options!(args)
+        pgroup_opt = Gem.win_platform? ? :new_pgroup : :pgroup
+
+        if args.last.is_a?(Hash)
+          args.last[pgroup_opt] = true
+        else
+          args.push(pgroup_opt => true)
+        end
+      end
+
+      def cleanup_process(pid)
+        Thread.new do
+          Process.kill('-TERM', pid)
```

Comment:
Oooh, of course.  Not sure why I didn't associate the `new_pgroup` code with the "kill the process group"... Carry on.